### DISCUSSION TITLE
SetterGeneral049 .. problem

### DISCUSSION THREAD

**Discussion by Jean-Gabriel1982** [GrandOrgue/GoOdf#71] [General]
Hello, ive built my organ and no problem. Just I decided at the end to select a stop in GO manual. I delete the stop in Manuals section, then Ranks section and it was deleted from the Panel. 

Then I save and open in GO. There is an error message and the organ is not loaded ..

Out of range value at section ´SetterGeneral049´ entry ´NumberOfStops' : 58

I tried to open my .organ file with text editor but all "Numberofstops" is correct ...

**Comment by larspalo** [comment]
Thanks for reporting! This is possibly a bug. But it could also simply be caused by the fact that a .cmb (settings file) and your .organ file is not "in sync" anymore after removing a stop. You could just try to save the organ again and see if that fixes the issue on subsequent loadings like suggested here https://github.com/GrandOrgue/grandorgue/discussions/1722#discussioncomment-7602739.

**Reply by Jean-Gabriel1982** [reply]
No I can't load the organ anymore it block the loading at start ..

I've recreated a stop in my GO manual in GOODF and my organ loads well after in GO.

**Reply by larspalo** [reply]
If it's really a problem with the settings file that was created when you saved an earlier version in GO, then it should be possible to go into File->Settings and under the Organs tab find the organ you've created, select it and then click the button below to remove the presets for that organ.

**Reply by Jean-Gabriel1982** [reply]
Ah ok doing this it works but delete all my settings .. it completely refreshes the organ but it works !
